tests/test_helpers.py

```diff
@@ -234,18 +234,32 @@ def test_name_with_import_error(self, modules_tmp_path):
 
 
 class TestStreaming:
-    def test_streaming_with_context(self, app, client):
-        @app.route("/")
-        def index():
+    def test_stream_with_context_fails_with_async_route(self):
+        import gc
```

Comment:
These imports should not be local to this function, and should not have spaces in between each one.